### PR TITLE
Add owner, createdBy, updatedBy to all securables and regen openapi docs

### PR DESCRIPTION
**Pull request by kriscon-db**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Added owner, createdBy and updatedBy to all securable info objects and database tables
Regenerated openAPI docs
Updated the id names for model versions and registered models

### CODE REVIEW

File: api/all.yaml [hunk 1]
```diff
@@ -1953,6 +2001,9 @@ components:
         comment:
           description: User-provided free-form text description.
           type: string
+        owner:
+          description: The user that owns the model
```
Comment (creechy) [inline api/all.yaml:2005]: nit: should this say "model version" ?

**Review by creechy — APPROVED** [review]
LGTM

File: api/all.yaml (hunk 1 above)
Comment (kriscon-db) [reply api/all.yaml:2005]: Yeah. Let me fix.

File: api/all.yaml [hunk 2]
```diff
@@ -1906,6 +1951,9 @@ components:
...
+          type: string
```
Comment (rameshchandra) [inline api/all.yaml:1956]: Why aren't there `created_by` and `updated_by` for models and model versions?

File: connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java [hunk 3]
```diff
@@ -321,14 +319,19 @@ public void testCreateExternalTableWithoutLocation() {
     SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
 
     String fullTableName1 = CATALOG_NAME + "." + SCHEMA_NAME + "." + PARQUET_TABLE;
-    assertThatThrownBy(() -> {
-      session.sql("CREATE EXTERNAL TABLE " + fullTableName1 + "(name STRING) USING parquet");
-    }).hasMessageContaining("Cannot create EXTERNAL TABLE without location");
+    assertThatThrownBy(
```
Comment (rameshchandra) [inline connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java:322]: The changes in this file are style changes. Can you remove it from the PR?

File: server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java [hunk 4]
```diff
@@ -21,25 +21,37 @@ public class CatalogInfoDAO extends IdentifiableDAO {
...
         .createdAt(
             catalogInfo.getCreatedAt() != null
                 ? Date.from(Instant.ofEpochMilli(catalogInfo.getCreatedAt()))
                 : new Date())
+        .createdBy(catalogInfo.getCreatedBy())
         .updatedAt(
             catalogInfo.getUpdatedAt() != null
                 ? Date.from(Instant.ofEpochMilli(catalogInfo.getUpdatedAt()))
                 : null)
+        .updatedBy(catalogInfo.getCreatedBy())
```
Comment (rameshchandra) [inline server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java:54]: Shouldn't this be `catalogInfo.getUpdatedBy()`?

File: server/src/main/java/io/unitycatalog/server/persist/dao/VolumeInfoDAO.java [hunk 5]
```diff
@@ -45,8 +54,11 @@ public VolumeInfo toVolumeInfo() {
         .name(getName())
         .comment(comment)
         .storageLocation(FileUtils.convertRelativePathToURI(storageLocation))
+        .owner(owner)
         .createdAt(createdAt.getTime())
+        .createdBy(createdBy)
         .updatedAt(updatedAt.getTime())
+        .updatedBy(updatedBy)
         .volumeType(VolumeType.valueOf(volumeType));
   }
```
Comment (rameshchandra) [inline server/src/main/java/io/unitycatalog/server/persist/dao/VolumeInfoDAO.java:64]: The `from` method below needs to be updated to include owner, createdBy, and updatedBy

File: api/all.yaml (hunk 2 above)
Comment (kriscon-db) [reply api/all.yaml:1956]: I had already added them previously.

File: connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java (hunk 3 above)
Comment (kriscon-db) [reply connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java:322]: I can try but testing might fail if the javafmt check fails.

File: server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java (hunk 4 above)
Comment (kriscon-db) [reply server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java:54]: Fixing.

File: server/src/main/java/io/unitycatalog/server/persist/dao/VolumeInfoDAO.java (hunk 5 above)
Comment (kriscon-db) [reply server/src/main/java/io/unitycatalog/server/persist/dao/VolumeInfoDAO.java:64]: Fixing.

**Review by rameshchandra — APPROVED** [review]
LGTM

**Pull request merged**